build-utils/peggy-loader.ts

```diff
@@ -0,0 +1,15 @@
+import peggy from 'peggy';
+
+export default function peggyLoader(source) {
+  // https://peggyjs.org/documentation.html#generating-a-parser-javascript-api
+  const peggyOptions: peggy.OutputFormatAmdCommonjsEs = {
```

Comment:
should we turn sourcemaps on? are those useful for people building .pegjs files

Reply:
I don't think it's very useful, it would be a source map for automatically-generated parser code that wouldn't make much sense to begin with